resources/sdk/purecloudjava/templates/ApiClient.mustache

```diff
@@ -975,4 +985,61 @@ public class ApiClient implements AutoCloseable {
         @Override
         public void close() throws Exception { }
     }
+
+     private ApiClientConnectorResponse retryInvoke(ApiClientConnectorRequest connectorRequest, ApiClientConnectorResponse connectorResponse) {
+         int retriesBeforeBackOff = 0;
+         long waitTimeInMillis;
+         int backOffRetries = 1;
+         long totalWaitTimeInMillis = 0L;
+         int MAX_RETRIES_BEFORE_BACKOFF = 5;
+         long FIVE_MINUTES_IN_MILLIS = 300000L;
+         long MAX_BACKOFF_IN_MILLIS = 27000L;
+         int MAX_BACKOFF_RETRIES = 3;
+
+         try {
+             do {
+                 if (connectorResponse.getHeaders().containsKey("Retry-After") && retriesBeforeBackOff++ < MAX_RETRIES_BEFORE_BACKOFF) {
+                     waitTimeInMillis = Integer.parseInt(connectorResponse.getHeaders().get("Retry-After")) * 1000L;
+
+                 } else if (backOffRetries > MAX_BACKOFF_RETRIES) {
+                     waitTimeInMillis = MAX_BACKOFF_IN_MILLIS;
```

Comment:
A 429 wouldn't need/what this max backoff retries applied. If the request is a 429, grab the retry-after value and invoke. The subsequent response could still be evaluated but the MAX_BACKOFF_RETRIES concept should be removed.